distributed/ddp-tutorial-series/multigpu_torchrun.py

```diff
@@ -22,6 +34,7 @@ def __init__(
         optimizer: torch.optim.Optimizer,
         save_every: int,
         snapshot_path: str,
+        device 
```

Comment:
would be nice to have type designation here:
```suggestion
        device: torch.device,
```